### PR TITLE
Add missing curly brace. Typo fix for a module import.

### PR DESCRIPTION
**Pull request by gburdeti**
Fixes #
- Change the variable name to *myValidator* to avoid a duplicate conflict - Modules Section.
- Typo fix for a module import using ```import =```.

### CODE REVIEW

File: pages/Symbols.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Just like strings, symbols can be used as keys for object properties.
 ```ts
 let sym = Symbol();
 
-let obj = {};
+let obj: any = {};
```
Comment (mhegazy) [inline pages/Symbols.md]: I would change it to:

```ts
let obj = {
   [sym]: "value"
};

console.log(obj[sym]); // "value"
```

Comment (DanielRosenwasser) [inline pages/Symbols.md]: I had the same idea actually, good call.

Comment (DanielRosenwasser) [inline pages/Symbols.md]: Only thing is that we never explained computer properties :smile:

**Comment by gburdeti** [comment]
@mhegazy made me realize that if *noImplicitAny* is set to false then the code works fine. I've set it in my tsconfig file. This kind of mistake won't happen again. Sorry.

**Comment by sandersn** [comment]
Actually, we try to have the handbook examples clean for `"noImplicitAny": true`. Can you re-add the `any` assertions in `Advanced Types.md`?

**Comment by sandersn** [comment]
After you add the casts back in, :+1:

**Comment by mhegazy** [comment]
:+1:

**Comment by gburdeti** [comment]
Then, I will reopen issue #239, because the code doesn't work unless *noImplicitAny* is false.

**Pull request merged**